Shared/Samples/Show service area/ShowServiceAreaView.swift

```diff
@@ -0,0 +1,222 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ShowServiceAreaView: View {
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    /// The data model for the sample.
+    @StateObject private var model = Model()
+    
+    /// Tracks whether to add facilities or barriers to map.
+    @State private var selectedGraphicType: SelectedGraphicType = .facilities
```

Comment:
@philium was this spacing intentional?

`  /// The currently selected graphic type.
    ///
    /// Used to track whether to add facilities or barriers to the map.`

and 

`@State private var selectedGraphicType: SelectedGraphicType = .facilities
    /// The error shown in the error alert.
    @State private var error: Error?`

Reply:
Yes. The extra line in the documentation differentiates the since sentence description from the discussion:

With Space | Without Space
--- | ---
![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/4b1b32b9-46a7-45d2-ba06-01262807db5e) | ![image](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/2257493/4164a269-63f4-4821-b888-777b0b109e85)

Without the extra line, there's no difference between this:

```swift
/// The currently selected graphic type.
/// Used to track whether to add facilities or barriers to the map.
```

And this:

```swift
/// The currently selected graphic type. Used to track whether to add
/// facilities or barriers to the map.
```

Regarding the lack of a blank line between state properties, that was intentional as well and explained in my original comment when I said to "use a blank line to separate groups." The state properties are one group, so they could all be together without blank lines in between, followed by a blank line, followed by the next group—a single state object property. In general, we do not separate stored properties with a blank line, especially for struts. That not only makes the declaration more succinct, with less needed whitespace, but it also allows for using a blank line to group stored properties together. Does that make sense?